### PR TITLE
Dp 8266  print styles v2

### PR DESCRIPTION
**Pull request by clair0917**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)

## Related Issue / Ticket

- [JIRA issue]()
- [Github issue]()

## Steps to Test

1. 

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

* 

#### @TODO

*

### CODE REVIEW

File: patternlab/styleguide/source/assets/scss/03-organisms/_top-actions.scss [hunk 1]
```diff
@@ -16,7 +16,7 @@
   &__items {
     @include clearfix;
 
-    @media print, ($bp-small-min) {
+    @media ($bp-small-min) {
       display: flex;
```
Comment (Tregonia) [inline patternlab/styleguide/source/assets/scss/03-organisms/_top-actions.scss:20]: The indention is off under this spot, but within this block.

File: patternlab/styleguide/source/assets/scss/08-print/_print.scss [hunk 2]
```diff
@@ -1,562 +1,432 @@
...
-    margin-bottom: 1.25rem;
-}
+  body {
+    color: $c-font-heading !important;
+  }
 
-.ma__rich-text__embed {
-  border: none;
-}
+  .ma__stacked-row-section .main-content .page-content>*,
```
Comment (Tregonia) [inline patternlab/styleguide/source/assets/scss/08-print/_print.scss]: Spaces around the `>*` generally help with readability of the code. Similar to the spaces around the `~` on line 24.

File: patternlab/styleguide/source/assets/scss/08-print/_print.scss [hunk 3]
```diff
@@ -1,562 +1,432 @@
...
+  .ma__banner-carousel,
+  .ma__quick-actions,
+  .ma__wait-time {
+    display: none !important;
   }
 
-  &:before {
-    display: none;
+  a::after {
+    content: " (" attr(href) ") ";
```
Comment (Tregonia) [inline patternlab/styleguide/source/assets/scss/08-print/_print.scss]: Are the spaces around the `()` required?
I defer to your thoughts on this one.

File: patternlab/styleguide/source/assets/scss/08-print/_print.scss [hunk 4]
```diff
@@ -1,562 +1,432 @@
...
+    .ma__stacked-row-section {
+      padding-top: 70px;
+      border-width: 0;
+    }
 
-  &__content {
-    min-height: 0;
-    padding: 0;
-    width: 100%;
+    .ma__stacked-row-section ~ .ma__stacked-row-section .ma__stacked-row-section__container:before {
```
Comment (Tregonia) [inline patternlab/styleguide/source/assets/scss/08-print/_print.scss]: Pseudo elements are identified with 2 colons `::`.

File: patternlab/styleguide/source/assets/scss/08-print/_print.scss [hunk 5]
```diff
@@ -1,562 +1,432 @@
...
+    }
 
-.ma__action-activities__details {
-  margin-left: 0;
-  width: 100%;
-}
+    .ma__page-banner__image {
+      display: none;
+      height: 0;
+      position: unset;
```
Comment (Tregonia) [inline patternlab/styleguide/source/assets/scss/08-print/_print.scss:176]: I could be wrong, but I do not believe that this is supported in IE11.

File: patternlab/styleguide/source/assets/scss/08-print/_print.scss [hunk 6]
```diff
@@ -1,562 +1,432 @@
...
-  padding-bottom: 30px;
-}
+  .ma__link-list__items {
+    
+    .ma__link-list__item {
+      width: 100%;
+      flex-basis: 100% !important;
+      display: block !important;
+
+        .ma__image-link__text {
```
Comment (Tregonia) [inline patternlab/styleguide/source/assets/scss/08-print/_print.scss]: This block is indented more than it should be.

File: patternlab/styleguide/source/assets/scss/base-theme.scss [hunk 7]
```diff
@@ -13,6 +13,9 @@
 @import "06-theme/00-base/fonts";
 @import "06-theme/00-base/pikaday";
 
+// Print styles
```
Comment (Tregonia) [inline patternlab/styleguide/source/assets/scss/base-theme.scss]: Comments should end in a full-stop period.

Comment (clair0917) [reply patternlab/styleguide/source/assets/scss/base-theme.scss]: The MASS coding standards don't require the period at the end of a comment

**Review by Tregonia — APPROVED** [review]

**Pull request merged**